app/src/main/java/ch/epfl/sweng/eventmanager/userManagement/InMemorySession.java

```diff
@@ -0,0 +1,34 @@
+package ch.epfl.sweng.eventmanager.userManagement;
+
+import javax.inject.Singleton;
+import ch.epfl.sweng.eventmanager.repository.data.User;
+
+/**
+ * This singleton class keeps the state of the current user.
+ */
+@Singleton
+public class InMemorySession {
+    private static InMemorySession instance = null;
```

Comment:
It works for now, in the future we might want to use @Inject directly

Reply:
Not a proper review, I close

Reply:
I'd like to keep things as simple as possible - and we don't need dependency injection here for now.